advocacy_docs/partner_docs/ToadEdgeGuide/03-SolutionSummary.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: 'Solution Summary'
+description: 'Brief explanation of the solution and its purpose'
+---
+
+Quest Toad Edge is a lightweight and reliable desktop database toolset that streamlines development and management tasks for EDB Postgres Advanced Server and EDB Postgres Extended. Its flexibility lies in it being built on Java and its ability to work with both Windows and Mac operating systems. Toad Edge supports coding, editing, schema compare and sync and DevOps CI processes, so you can manage your databases easily and confidently.         
+
```

Comment:
processes, to you manage EDB Postgres Advanced Server and EDB Postgres Extended.